packages/commonwealth/server/util/requirementsModule/validateGroupMembership.ts

```diff
@@ -20,7 +20,7 @@ export default function validateGroupMembership(
   userAddress: string,
   requirements: Requirement[],
   tbc?: TokenBalanceCache
-): validateGroupMembershipResponse {
+): ValidateGroupMembershipResponse {
```

Comment:
Update to `Promise<validateGroupMembershipResponse>` to avoid issues with merge order/missing the `await` in the refreshMembership code.